### ISSUE TITLE
not working on fedora 40 kde

### ISSUE THREAD

**Issue by siam78** [OpenBangla/OpenBangla-Keyboard#382]
Unknown option 'show-progress'
![Screenshot_20240507_120223](https://github.com/OpenBangla/OpenBangla-Keyboard/assets/55010104/39e40a2f-29fd-484d-9ddc-c931fca067f9)

**Comment by kaytiseprofessor** [comment]
GNU Wget2 2.1.0 - multithreaded metalink/file/website downloader

+digest +https +ssl/gnutls +ipv6 +iri +large-file +nls -ntlm -opie +psl -hsts
+iconv +idn2 +zlib -lzma +brotlidec +zstd -bzip2 -lzip +http2 +gpgme

Copyright (C) 2012-2015 Tim Ruehsen
Copyright (C) 2015-2021 Free Software Foundation, Inc.

License GPLv3+: GNU GPL version 3 or later
<http://www.gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Please send bug reports and questions to <bug-wget@gnu.org>.

**Comment by butter-fly04** [comment]
Same here in Fedora 40 Gnome

**Comment by Onxr** [comment]
Fedora 40 KDE
Same here. Had this error so I just downloaded f38 version, installed using `dnf`
Now from app launcher I can't find iBus preference. I tried running `ibus start &` in terminal then got an icon at the system tray.
But I can only change once. If I want to change input type again I have to logout and login. Also if I want to exit using `ibus exit` it loops like if I am holding `Ctrl+c` but doesn't show `^C` in the terminal

I also tried in x11 session. This time I found iBus preference in app launcher and also was able to switch but bangla typing didn't work. Logged out and logged in to wayland same issue. Bangla typing doesn't work.
Restarted the machine, logged in a wayland session, now once again I cant switch input method without a logout then logging in.

**Comment by tazihad** [comment]
I made a COPR for fedora
https://copr.fedorainfracloud.org/coprs/badshah/openbangla-keyboard/

For KDE wayland. You must enable ibus-wayland from virtual keyboard.

**Comment by butter-fly04** [comment]
Does this work for Fedora 40?

Asking this because I was guessing the installation problem was due to the absence of the .rpm package for Fedora 40.

**Comment by Onxr** [comment]
> Does this work for Fedora 40?
> 
> Asking this because I was guessing the installation problem was due to the absence of the .rpm package for Fedora 40.

Yes its working but you will also need `ibus-openbangla`. I thought I didn't need it. But without it I wasn't able to write in Bangla. After installing `ibus-openbangla` I got the `ibus-wayland` option in virtual keyboard. Looked similar so I went to the input method configuration page from the tray icon and added `open bangla` there.
After these uninstalled (`ibus-openbangla`) after configuring it and it did work for me (I was trying to test). BTW  then I set `Fcitx 5 wayland launcher` instead of `IBus Wayland`.
Everything looks fine. I still have the `IBus Wayland` option even though I haven't that before and I have uninstalled `ibus-openbangla`.
